datadog_checks_dev/datadog_checks/dev/tooling/utils.py

```diff
@@ -251,6 +251,12 @@ def get_metadata_file(check_name):
     return os.path.join(get_root(), check_name, 'metadata.csv')
 
 
+def get_eula_from_manifest(check_name):
```

Comment:
Can we use the `get_assets_from_manifest` util? I think the only difference is that it returns list of paths vs 1 path in `get_eula_from_manifest`.

Reply:
The eula isn't under assets key, its under its own `terms` key. The `get_assets_from_manifest` hardcodes in `assets`, so I figured a new util would be needed here.